gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,582 @@
+
+  .. literalinclude:: ./manifests/input/kubeadm-init-config.yaml
+    :language: yaml
+    :caption: Sample Kubeadm Init Config with DRA Feature Gates Enabled
+
+- The NVIDIA GPU Operator v25.3.0 or later installed with CDI enabled on all nodes and NVIDIA GPU Driver 565 or later.
+  
+  A sample Helm install command below includes enabling CDI with ``cdi.enabled=true``.
+  Refer to the install documentation for details on `enabling CDI <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#common-chart-customization-options>`__.
+
+  .. code-block:: console
+
+      $ helm install --wait --generate-name \
+              -n gpu-operator --create-namespace \
+              nvidia/gpu-operator \
+              --version=${version} \
+              --set cdi.enabled=true
+
+  If you want to install the DRA Driver for GPUs using pre-installed drivers, you must install NVIDIA GPU Driver 565 or later, the corresponding IMEX packages on GPU nodes, and disable the IMEX systemd service before installing the GPU Operator.
+  Refer to the documentation on `installing the GPU Operator with pre-installed drivers <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#pre-installed-nvidia-gpu-drivers>`__ for more details.
+
+  NVIDIA DRA Driver for GPUs also requires the NVIDIA Container Toolkit (nvidia-ctk) v1.17.5 or later, which is installed by default with the NVIDIA GPU Operator v25.3.0 and later.
+  Refer to the `NVIDIA Container Toolkit documentation <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html>`__ for installation instructions. 
+
+Install the NVIDIA DRA Driver for GPUs with Helm
+================================================
+
+#. Add the NVIDIA Helm repository:
+
+   .. code-block:: console
+
+      $ helm repo add nvidia https://helm.ngc.nvidia.com/nvidia \
+          && helm repo update
+
+#. Install NVIDIA DRA Driver:
+
+   .. code-block:: console
+
+      $ helm install nvidia-dra-driver-gpu nvidia/nvidia-dra-driver-gpu \
+            --version="25.3.0" \
+            --create-namespace \
+            --namespace nvidia-dra-driver-gpu \
+            --set nvidiaDriverRoot=/run/nvidia/driver \
+            --set nvidiaCtkPath=/usr/local/nvidia/toolkit/nvidia-ctk \
+            --set resources.gpus.enabled=false
+
+Common Chart Customization Options
+==================================
+
+The following options are available when using the Helm chart.
+These options can be used with ``--set`` when installing with Helm.
+
+The following table identifies the most frequently used options.
+To view all the options, run ``helm show values nvidia/nvidia-dra-driver-gpu``.
+
+.. list-table::
+   :widths: 20 50 30
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+     - Default
+
+   * - ``nvidiaDriverRoot``
+     - Specifies the driver root on the host.
+       For Operator-managed drivers (recommended), use ``/run/nvidia/driver``.

```

Comment:
For clarity, lets be specific since our readers may not know what "operator" we are referring to.
```suggestion
       For GPU Operator-managed drivers (recommended), use ``/run/nvidia/driver``.

```